sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -27,6 +30,14 @@ export interface DefaultAzureCredentialOptions extends TokenCredentialOptions {
    * This client ID can also be passed through to the {@link ManagedIdentityCredential} through the environment variable: AZURE_CLIENT_ID.
    */
   managedIdentityClientId?: string;
+  /**
+   * Optionally pass in a resource ID to be used by the {@link ManagedIdentityCredential}.
+   * In scenarios such as when user assigned identities are created using an ARM template,
+   * where the resource Id of the identity is known but the client Id can't be known ahead of time,
```

Comment:
Nit: In prose, "ID" is usually written in all caps even though it is an abbreviation for "identifier" in this case.

Reply:
I went and looked at the frequency of occurrence in the identity package:

(using a command like `grep -r "\sID\s" ./**/.md ./src/**/*.ts | wc -l`)

`ID`: 67
`id`: 8
`Id`: 8

It's not a big deal, but any time ID is written another way in prose it sticks out to me.